src/celeritas/em/distribution/MuBremsPPAngularDistribution.hh

```diff
@@ -0,0 +1,103 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/distribution/MuBremsPPAngularDistribution.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/math/Algorithms.hh"
+#include "celeritas/Constants.hh"
+#include "celeritas/Quantities.hh"
+#include "celeritas/random//distribution/GenerateCanonical.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Sample the polar angle for muon bremsstrahlung and pair production.
```

Comment:
Nice docs, can you include the class in the `Bremsstrahlung` section of the `em-physics` section?